docker-compose.yaml

```diff
@@ -67,8 +76,54 @@ services:
       - BENTO_PUBLIC_INTERNAL_PORT
       - BENTO_BEACON_CONTAINER_NAME
       - BENTO_BEACON_INTERNAL_PORT
+      - BENTO_CBIOPORTAL_CONTAINER_NAME
+      - BENTO_CBIOPORTAL_INTERNAL_PORT
     networks:
-      bridge-net:
+      aggregation-net:
+        aliases:
+          - ${BENTOV2_DOMAIN}
+          - ${BENTOV2_PORTAL_DOMAIN}
+      auth-net:
+      beacon-net:
+        aliases:
+          - ${BENTOV2_DOMAIN}
+          - ${BENTOV2_PORTAL_DOMAIN}
+      cbioportal-net:
+      drop-box-net:
+        aliases:
+          - ${BENTOV2_DOMAIN}
+          - ${BENTOV2_PORTAL_DOMAIN}
+      drs-net:
+        aliases:
+          - ${BENTOV2_DOMAIN}
+          - ${BENTOV2_PORTAL_DOMAIN}
+      event-relay-net:
```

Comment:
No aliases here?

Reply:
i'm pretty sure they're not needed because event relay doesn't have to talk to the gateway (1 way traffic)